deployment/cdk/opensearch-service-migration/lib/stack-composer.ts

```diff
@@ -190,19 +190,17 @@ export class StackComposer {
         }
 
         // Currently, placing a requirement on a VPC for a historical capture stack but this can be revisited
-        // Note: Future work to provide orchestration between historical capture and migration assistance as the current
-        // state will potentially have both stacks trying to add the same data
+        // TODO: Future work to provide orchestration between historical capture and migration assistance
         if (historicalCaptureEnabled && networkStack) {
             const historicalCaptureStack = new HistoricalCaptureStack(scope, "historicalCaptureStack", {
                 vpc: networkStack.vpc,
-                logstashConfigFilePath: logstashConfigFilePath,
+                dpPipelineTemplatePath: dpPipelineTemplatePath,
                 sourceEndpoint: sourceClusterEndpoint,
-                targetEndpoint: opensearchStack.domainEndpoint,
+                targetEndpoint: process.env.MIGRATION_DOMAIN_ENDPOINT!,
```

Comment:
Was not expecting to get the target endpoint like this (do we ever add this to the current CDK environment?), any issue with using `opensearchStack.domainEndpoint` here

Reply:
`process.env.` is used to access environment variables so this is accessing the `MIGRATION_DOMAIN_ENDPOINT` variable that is included in the networkStack's export commands. 

I believe i moved away from `opensearchStack.domainEndpoint` because this variable isn't available/populated when no changes are made to the openSearchStack.

The ideal fix here would be to use CFN Output variables. I'll tackle that in a follow up PR.

Reply:
> I'd be very surprised if opensearchStack.domainEndpoint wasn't available on every deployment (even with no changes) as I have been testing this way for a while. Happy to see this in action though and get it resolved if that is the case.

@lewijacn you're probably right. I now recall why i made this change, and it doesn't have to do with the CDK stack. Rather, using the environment variable decouples the HistoricalMigration stack from the creation of the `opensearchStack` via the CDK. This way, a user can plug in an existing stack as the target by simply setting the `MIGRATION_DOMAIN_ENDPOINT` environment variable.

If you'd prefer I revert to `opensearchStack.domainEndpoint`, let me know - i'll add a dependence in the surrounding if-condition: https://github.com/kartg/opensearch-migrations/blob/d7c2cb1fff42ad3bb19ea5e5a68fe32e6147ef5d/deployment/cdk/opensearch-service-migration/lib/stack-composer.ts#L194

Reply:
I wouldn't worry about that too much right now, as this will get overhauled by the Parameter store changes. I was more concerned with the other comments you had, but glad to hear these are not an issue 👍